backend/tests/test_schema/helpers.py

```diff
@@ -32,12 +29,15 @@ def upgrade(self, migration_id: str) -> str:
             f"DATABASE_URL={self.engine.url} "
             f"alembic upgrade {migration_id}"
         )
-        out = check_output(cmd, shell=True, stderr=STDOUT, cwd=cwd).decode("utf-8")
         print(
             f"""--- Alembic upgrade ---
-        |CWD: {cwd}
+        |CWD: {os.getcwd()}
         |CMD: {cmd}
-        |STDOUT:
+        """
+        )
+        out = check_output(cmd, shell=True, stderr=STDOUT).decode("utf-8")
```

Comment:
This was quite useful - the logging of the command and cwd is logged before the actual running of the command - and the output after